cupyx/scipy/special/_expn.py

```diff
@@ -0,0 +1,359 @@
+# This source code contains SciPy's code.
+# https://github.com/scipy/scipy/blob/main/scipy/special/cephes/expn.c
+#
+#
+# Cephes Math Library Release 1.1:  March, 1985
+# Copyright 1985 by Stephen L. Moshier
+# Direct inquiries to 30 Frost Street, Cambridge, MA 02140
+
+from cupy import _core
+
+from cupyx.scipy.special._gamma import gamma_definition
+
+
+polevl_definition = '''
+
+static __device__ double polevl(double x, const double coef[], int N)
```

Comment:
Removing `const` qualifier here is fairly acceptable as this function is short and clear enough.

```suggestion
static __device__ double polevl(double x, double coef[], int N)
```

Reply:
Updated :)